cmd/run/run.go

```diff
@@ -217,7 +217,11 @@ func (c *Command) ensureAsyncPreemptionHack() error {
 		environ = append(environ, "GODEBUG=asyncpreemptoff=1")
 		environ = append(environ, "SUBTRACE_ORIG_GODEBUG=<empty>")
 	}
-	if err := unix.Exec(os.Args[0], os.Args, environ); err != nil {
+	abspath, err := exec.LookPath(os.Args[0])
+	if err != nil {
+		return fmt.Errorf("lookpath: %w", err)
+	}
+	if err := unix.Exec(abspath, os.Args, environ); err != nil {
```

Comment:
I think this should mostly work, but `os.Executable` is probably more robust. For instance, if the subtrace executable got executed without its args[0] being in `$PATH`, using `os.Executable` will still return a valid path. Admittedly a very rare scenario, but it's good to be super defensive this early in the process' life.

Reply:
Agreed, we can't guarantee that the executable is in `$PATH` , in fact that's a good scenario for me to test separately anyway

Reply:
@adtac Should we make the same change in the `entrypointChild` function in this file? It currently uses `exec.LookPath`

Reply:
That's looking for the user's binary, not for subtrace's own binary. We still need LookPath there.